proxy/frontend/type/postgresql/src/main/java/org/apache/shardingsphere/proxy/frontend/postgresql/command/query/extended/parse/PostgreSQLComParseExecutor.java

```diff
@@ -61,6 +62,11 @@ public Collection<DatabasePacket> execute() {
         SQLParserEngine sqlParserEngine = createShardingSphereSQLParserEngine(connectionSession.getDatabaseName());
         String sql = packet.getSQL();
         SQLStatement sqlStatement = sqlParserEngine.parse(sql, true);
+        String escapedSql = escape(sqlStatement, sql);
+        if (!escapedSql.equalsIgnoreCase(sql)) {
+            sqlStatement = sqlParserEngine.parse(escapedSql, true);
```

Comment:
Is `??&` a valid PostgreSQL syntax? If we escape `?` here, maybe parsing SQL contains `??&` should be failed?

Reply:
Ignore it. Our PostgreSQL parser has already treated `?` as valid syntax.